src/lib.rs

```diff
@@ -40,7 +45,7 @@ pub enum AccountType
 }
 
 /// The current status of a price feed.
-#[derive(Copy, Clone, PartialEq)]
+#[derive(Copy, Clone, PartialEq, BorshSerialize, BorshDeserialize, Debug)]
```

Comment:
As far as I can see Borsh serializations aren't used anywhere in the PR, do we need these?

Reply:
It's used in the instruction.rs for testing purposes.

Reply:
Is it clear now? @Reisen @jayantk